.gitignore

```diff
@@ -5,6 +5,7 @@ data/
 log/
 #Ignore all application.properties file that might contain data from local device
 app/src/main/resources/**/application*.properties
+distribution/sender/old_routes/
```

Comment:
What is this for?

Reply:
I put on this folder a backup of original routes just for myseft consultation. I'll remove after I done with routes review

Reply:
You can always find them in the git history, so you don't need copy them anywhere else

Reply:
Did you see the comment above?

Reply:
Addressed